cognite/powerops/custom_modules/power_model_v1/data_models/containers/ProductionObligationForBenchmarkingDayAhead.container.yaml

```diff
@@ -0,0 +1,22 @@
+space: "{{power_ops_models}}"
+externalId: ProductionObligationForBenchmarkingDayAhead
+name: ProductionObligationForBenchmarkingDayAhead
```

Comment:
My comment is just that we should try to not have infinite iterations on this, because we have a deadline here.

Reply:
Naming is still very relevant since the SDK and all graphql queries are dependent on using these names. Hence why a standard pattern of naming should be used to simplify usage by end users.